packages/ui/components/form/selectimproved/components/SearchInput.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext } from "react";
+
+import { FiSearch } from "../../../icon";
+import { SelectContext } from "./SelectProvider";
+
+interface SearchInputProps {
+  placeholder?: string;
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  name?: string;
+  searchInputRef?: React.RefObject<HTMLInputElement>;
+}
+
+const SearchInput: React.FC<SearchInputProps> = ({
+  placeholder = "",
+  value = "",
+  onChange,
+  name = "",
+  searchInputRef,
+}) => {
+  const { classNames } = useContext(SelectContext);
+  return (
+    <div
+      className={
+        classNames && classNames.searchContainer ? classNames.searchContainer : "relative py-1 px-2.5"
+      }>
+      <FiSearch
+        className={
+          classNames && classNames.searchIcon
+            ? classNames.searchIcon
+            : "absolute mt-2.5 ml-2 h-5 w-5 pb-0.5 text-gray-500"
+        }
+      />
+      <input
```

Comment:
This had different styles to our normal input

Wasnt worth building a new variant as this is probably one of a kind and didnt need the added RHF logic that comes with our input